code/modules/vending/clothesmate.dm

```diff
@@ -137,6 +137,13 @@
 		            /obj/item/clothing/shoes/cowboy = 2,
 		            /obj/item/clothing/shoes/cowboy/white = 2,
 		            /obj/item/clothing/shoes/cowboy/black = 2,
+		            //SKYRAT EDIT: ADDITION: START - Footwraps
```

Comment:
I thought it's BEGIN instead of START..?

Reply:
Everything in the clothesmate uses start, so I just... went with it.  🤷 I can change them all to begin if preferred

Reply:
Less about "preferred", more just confused about how is it actually supposed to be.